orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestForceCacheRefreshTask.java

```diff
@@ -288,6 +288,8 @@ private StageData fromStage(Stage stage) {
     Set<ScopedManifest> processedManifests = new HashSet<>();
 
     Set<String> errors = new HashSet<>();
+
+    Map<String, List<String>> manifestNamesByNamespaceToRefresh = null;
```

Comment:
Ugh sorry I'm breaking the tests, I guess I can't just set this to null here...do you know if there is a reason this task is using a static class and its own special `fromStage` method instead of `stage.mapTo`? Will fix this up on Tuesday :sob: